handsontable/src/plugins/exportFile/exportFile.js

```diff
@@ -44,6 +45,42 @@ export const PLUGIN_PRIORITY = 240;
  *   range: [1, 1, 6, 6]         // [startRow, endRow, startColumn, endColumn]
  * });
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ * />
+ *
+ * // access to exportFile plugin instance
+ * const exportPlugin = hotRef.getPlugin('exportFile');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae